app/assets/javascripts/collections/story_collection.js

```diff
@@ -18,25 +18,61 @@ module.exports = Backbone.Collection.extend({
     this.labels = [];
   },
 
+  saveSorting: function(columnName) {
```

Comment:
The spacing on these functions is inconsistent. You have functions with and without an empty space before the curly bracket.
To keep the consistency and following a [more standard](https://github.com/airbnb/javascript#functions--signature-spacing) styleguide, prefer to use the empty space.
```js
function (args) {
  //function body
}
```

Also, is important to know what is the project's styleguide pattern and follow it. 💅